src/models/userModel.js

```diff
@@ -0,0 +1,47 @@
+const mongoose = require("mongoose");
+const bcrypt = require("bcrypt");
+
+const Schema = mongoose.Schema;
+
+let userSchema = new Schema({
```

Comment:
We would prefer a `const`, check the eslint config it should tell you normally

Reply:
done